src/test/java/uk/gov/hmcts/reform/sendletter/logging/AppInsightsTest.java

```diff
@@ -49,10 +61,77 @@ public void should_fail_creating_app_insights_when_instrumentation_key_is_not_pr
 
     @Test
     public void should_create_app_insights_with_default_dev_mode_off() {
-        context.setInstrumentationKey("some-key");
+        context.setInstrumentationKey(IKEY);
 
         new AppInsights(telemetry);
 
         assertThat(TelemetryConfiguration.getActive().getChannel().isDeveloperMode()).isFalse();
     }
+
+    @Test
+    public void should_track_message_acknowledgement_event_for_success_case() {
+        context.setInstrumentationKey(IKEY);
+
+        AppInsights insights = new AppInsights(telemetry);
+        Map<String, String> properties = Collections.singletonMap("messageId", MESSAGE_ID);
+        Duration timeTook = Duration.ofMinutes(1);
+
+        insights.trackMessageAcknowledgement(timeTook, true, MESSAGE_ID);
+
+        verify(telemetry).trackDependency(
+            AppInsights.SERVICE_BUS_DEPENDENCY,
+            AppInsights.SERVICE_BUS_MESSAGE_ACKNOWLEDGED,
+            new com.microsoft.applicationinsights.telemetry.Duration(timeTook.toMillis()),
+            true
+        );
+        verify(telemetry).trackEvent(AppInsights.SERVICE_BUS_MESSAGE_ACKNOWLEDGED, properties, null);
+    }
```

Comment:
Are you sure there are no more interactions ?It would be good if we add `verifyNoMoreInteractions(telemetry);`